### PR TITLE
2.0 forsetisecurity.org dev preseeded news articles 

### PR DESCRIPTION
**Pull request by jjawad-google**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [x] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Forseti 2.0 Launch
+author: Henry Chang
+---
+Forseti 1.0 was initially launched at Google Cloud Next '17 a little over a year ago. Since then, Forseti went on to be adopted by over 100 businesses and organizations, helping to secure Google Cloud Platform environments large and small.
+
+We received great feedback on how Forseti could be improved, as well as insight into additional features you like to see, which require making fundamental architectural changes to Forseti in order to make the implementation possible.  The Forseti team embarked on a journey to make these changes, in order to take Forseti to the next level.  Thanks to hard work with many contributors, we are excited to share with you some of the exciting new changes in Forseti 2.0 below!
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: Overall comments:
- Please keep your character line length to 80, if that causes wrapping on a URL that's fine.
- One space after a period

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<add links for each of the items to website; what is the website url?>
+
+New Features
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: **New Features** for it to be bolded.

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* New Explain component to help you understand the effective roles and permissions that affect your Google Cloud Platform resources from the granted IAM policies.
+* New CLI client to operate all the Forseti components on demand.
+* Parallelized inventory process.
+* Easier to add new inventory types : Bucket IAM policies, API configurations, audit logging configurations, networks, subnetworks, and G Suite Users.
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: nit: Remove the space before the colon

Comment (mirons-google) [reply _news/2018-06-07-forseti-2.0-launch.md]: Your list of things has mixed case patterns, just standardize it.

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* New scanners for service account key rotation, audit log configuration, and enabled APIs.
+* IAM policy scanner now audits bucket IAM policies.
+* A more intuitive installation process with better default rule files.
+
+Architecture Improvements
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: **Architecture Improvements**

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Centralized table for all GCP inventory data, with optimized schema and indexes for easier and faster queries.
+* New auth library that negates the use of a separate G Suite service account and exported key.
+* Forseti GCE VMs now run on the latest Ubuntu 18.04.
+
+If you are interested to upgrade from version 1.0, please see this
+[guide](http://add
+links to upgrade guide here) 
+
+If you are new to Forseti, please see this [guide](http://add links to
+getting started guide) to get started.
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: @joecheuk should have this link.

File: _news/2018-06-07-forseti-2.0-launch.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+* IAM policy scanner now audits bucket IAM policies.
+* A more intuitive installation process with better default rule files.
+
+Architecture Improvements
+* Centralized table for all GCP inventory data, with optimized schema and indexes for easier and faster queries.
+* New auth library that negates the use of a separate G Suite service account and exported key.
+* Forseti GCE VMs now run on the latest Ubuntu 18.04.
+
+If you are interested to upgrade from version 1.0, please see this
+[guide](http://add
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-2.0-launch.md]: @joecheuk should have this link

File: _news/2018-06-07-forseti-hackday.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+title:  Forseti Hack Day
+author: Justin Jawad
+---
+Google is hosting the first Forseti Security Hack Day on July 23rd. Thank you to all who registered! We have had a very positive international response from participants representing leading organizations.
+
+Come hungry! We have a full agenda planned plus breakfast / lunch a la Google with speakers presenting on a variety of cloud security-related topics. Dive into Forseti with us and learn more about how to :
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-hackday.md]: nit: put Dive into on a line by itself and remove the use of the colon.

File: _news/2018-06-07-forseti-hackday.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+* Keep track of your environment.
+* Monitor and understand your current Google Cloud Platform security policies.
+* Provide corrective action when problems are detected.
+
+Spend the day with the Forseti team at Google, gain insight into what’s on the horizon, and help make our code base stronger together!
+
+We still have openings but they won’t last long! Register now to attend.
+
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-hackday.md]: Register should be a link.

Comment (mirons-google) [inline _news/2018-06-07-forseti-hackday.md]: Nit: You can remove the trailing newline.

File: _news/2018-06-07-published-gcp-podcast.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+---
+title: Google Cloud Platform Podcast
+author: Andrew Hoying and Nenand Stojanovski
+---
+Andrew Hoying, Senior Security Engineer - Google and Nenand Stojanovski, Staff
+Security Engineer - Spotify join Mark Mandel & Melanie Warrick on the weekly Google Cloud Platform Podcast to talk Forseti, GCP Security and everything in between. Tune in [here](https://www.gcppodcast.com/post/episode-120-forseti-with-nenad-stojanovski-and-andrew-hoying/)!
+
+Like what you hear? [Subscribe](https://feeds.feedburner.com/GcpPodcast) to the Google Cloud Platform Podcast.
+
```
Comment (mirons-google) [inline _news/2018-06-07-published-gcp-podcast.md]: nit: remove trailing newline

File: _news/2018-06-07-published-gcp-seurity-best-practices.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+---
+title: GCP Security Best Practices
+author: Justin Jawad
+---
+Forseti Security provides security assurance for your GCP environment. Check out the links below to learn more about best practices around securing GCP environments from the team @ Google. 
+
+* [IAM Security Best Practices](https://cloud.google.com/iam/docs/using-iam-securely)
+
```
Comment (mirons-google) [inline _news/2018-06-07-published-gcp-seurity-best-practices.md]: Ah, I was hoping we could introduce our own best practices (/concepts/best-practices.html) here. Can you rewrite that to do this? Maybe in that case we can make @MartenvanWezel the author as he's updating it.

**Comment by jjawad-google** [comment]
Changes pushed, ready for review.

**Comment by jjawad-google** [comment]
Thank you @Tabrarian . @mirons-google are we clear to merge? There are some required cla and travis checks that are currently failing / pending.

File: _news/2018-06-07-forseti-hackday.md [hunk 11]
```diff
@@ -2,14 +2,19 @@
...
 * Monitor and understand your current Google Cloud Platform security policies.
 * Provide corrective action when problems are detected.
 
-Spend the day with the Forseti team at Google, gain insight into what’s on the horizon, and help make our code base stronger together!
-
-We still have openings but they won’t last long! Register now to attend.
+Spend the day with the Forseti team at Google, gain insight into what’s on the
+horizon, and help make our code base stronger together!
 
+We still have openings but they won’t last long! [Register for Forseti Hack Day](https://docs.google.com/forms/d/1QzJHCTfG0_E4ag7NMk5e6LwVNwn81I7P-HDqJygeijg/edit) now to attend.
```
Comment (mirons-google) [inline _news/2018-06-07-forseti-hackday.md]: This link is probably not the want you want to use? If it's a form there's a menu option in the form to get a link to the form, use that one instead.

**Comment by mirons-google** [comment]
Just a fix on the URL and one nit to complete before merging. Also, did you chose not to backdate the articles? The date of the article comes from the title of the file.

**Comment by jjawad-google** [comment]
@mirons-google : Hack Day registration form link updated. File names of posts backdated to match original content posting dates.

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as all commit authors have contributed to this project before.

**Pull request merged**